src/Components/WebAssembly/WebAssembly/src/Infrastructure/JSInteropMethods.cs

```diff
@@ -1,19 +1,32 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.ComponentModel;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Components.RenderTree;
 using Microsoft.AspNetCore.Components.Web;
+using Microsoft.AspNetCore.Components.WebAssembly.Rendering;
+using Microsoft.AspNetCore.Components.WebAssembly.Services;
 using Microsoft.JSInterop;
 
-namespace Microsoft.AspNetCore.Components.WebAssembly.Rendering
+namespace Microsoft.AspNetCore.Components.WebAssembly.Infrastructure
```

Comment:
While we agreed on `.Internal`, I think we're just going to get devs on this team pestering us about that particular choice of the namespace. We use Infrastructure to part other utility-type code, so it would be less problematic. If you feel strongly about it, I can change it to use `.Internal`.

Reply:
OK, thanks for calling it out. I think it's fine like this.